Singularity_debug.pandora

```diff
@@ -41,16 +41,6 @@ MirrorURL: http://mirrors.ukfast.co.uk/sites/archive.ubuntu.com/
     cd ..
 
```

Comment:
Change the name of the file to be `Singularity.pandora_debug` otherwise the tags on Singularity Hub will overwrite the other pandora recipe.